README.md

```diff
@@ -99,14 +99,14 @@ Details about the API are available in the [`docs`](docs) directory.
 <a name="Prerequisites"></a>
 ## Prerequisites
 
-The aerospike package supports Node.js v0.12.x, v4.2.x (LTS) and v5.x as well
+The aerospike package supports Node.js v4.x (LTS), v6.x (LTS) and v8.x (LTS) as well
```

Comment:
Thanks for pointing out the conflicting version information listed in this section! I had missed this.